src/compiler/commandLineParser.ts

```diff
@@ -275,9 +280,9 @@ module ts {
     }
 
     /**
-      * Read tsconfig.json file
-      * @param fileName The path to the config file
-      */
+     * Read tsconfig.json file
+     * @param fileName The path to the config file
+     */
     export function readConfigFile(fileName: string): any {

```

Comment:
Never mind, I just scrolled down. :smile: 

So do we really need to expose this?